doc/source/security/encrypt-traffic.rst

```diff
@@ -42,73 +49,84 @@ for each node:
    ssl-cert=/etc/mysql/certs/client-cert.pem
    ssl-key=/etc/mysql/certs/client-key.pem
 
-After restart the node will use these files to encrypt communication with
-clients. MySQL clients require only the second part of the configuration
-to communicate with cluster nodes.
-
-|MySQL| generates default key and certificate
-files and places them in data directory. You can either use them or generate
-new certificates. For generation of new certificate please refer to
-:ref:`generate-keys-certs` section.
+After it is restarted, the node uses these files to encrypt communication with
+clients. |MySQL| clients require only the second part of the configuration to
+communicate with cluster nodes.
 
 .. _encrypt-replication-traffic:
 
 Encrypting Replication Traffic
 ==============================
 
-Replication traffic refers to the inter-node traffic which includes
-:term:`SST` traffic, :term:`IST` traffic, and replication traffic.
+*Replication traffic* refers to the inter-node traffic which includes
+the :term:`SST` traffic, :term:`IST` traffic, and replication traffic.
 
-Traffic of each type is transferred via different channel, and so it is
-important to configure secure channels for all 3 variants to completely
-secure the replication traffic.
+The traffic of each type is transferred via a different channel, and so it
+is important to configure secure channels for all 3 variants to
+completely secure the replication traffic.
 
-|PXC| supports a single configuration option which helps to
-secure complete replication traffic, and is often referred as Automatic
-Configuration. User can also ignore this and configure security of
-each channel by specifying independent parameters.
-
-Section below will help, covering this aspect.
+|PXC| supports a single configuration option which helps to secure the complete
+replication traffic, and is often referred to as :ref:`ssl-auto-conf`. You can
+also configure the security of each channel by specifying independent
+parameters.
 
 .. _ssl-auto-conf:
 
 SSL Automatic Configuration
 ===========================
 
+The automatic configuration of the SSL encryption needs a key and certificate
+files. MySQL generates a default key and certificate files and places them in
+the data directory.
+
+.. important::
+
+   It is important that your cluster use the same SSL certificates on all nodes.
+ 
```

Comment:
Note that encryption can't be enabled on a running cluster. If it was disabled before the cluster was bootstrapped, cluster needs to stopped, encryption set up, and bootstrapped again.